src/main/scala/com/stackstate/pac4j/AkkaHttpSecurity.scala

```diff
@@ -66,41 +68,73 @@ class AkkaHttpSecurity(config: Config, sessionStorage: SessionStorage)(implicit
   import AkkaHttpSecurity._
   // TODO: At some point this object should contain the SessionStore (when we implement that)
```

Comment:
Remove this comment, session store is in place.

Reply:
done